pages/management/devices.md

```diff
@@ -78,6 +87,18 @@ On all resin.io devices `/data` is a persistent data volume. This is useful for
 
 It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of it.
```

Comment:
Maybe also add the "version available" note here too, as you did with the app action?

``__Warning:__ This action is only supported on devices with an Agent version >= 1.1.0``